doc/user-guide/MongooseIM-High-level-Architecture.md

```diff
@@ -46,6 +46,8 @@ A HTTP notification enables the forwarding of events to any external HTTP servic
 
 WombatOAM enables the monitoring and management of MongooseIM clusters, as well as Riak KV, RabbitMQ, and any other Erlang and Elixir based system.
 
+![MongooseIM high-level architecture](MongooseIM_high-level_architecture.png)
```

Comment:
> shouldn't be there some connections (e.g lines) between mongooseim cluster and databases?

That's true, some lines to **_connect the dots_** as it were, might help bring conceptual clarity to the diagram. At the moment we have pieces but it's not clear how they fit together. Perhaps we can line the DBs up in a row alongside the HTTP box too.